migrations/20240731135626-fix-migration-replacement.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var fs = require('fs');
+var path = require('path');
+var bcrypt = require('bcrypt');
+var dotenv = require('dotenv');
+var util = require('util'); // Import the util module
+
+dotenv.config();
+
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+  Promise = options.Promise;
+};
+
+exports.up = function(db) {
+  // Using promisified readFile
+  const readFileAsync = util.promisify(fs.readFile); // Use util.promisify
+  const filePath = path.join(__dirname, 'sqls', '20240731132502-initialize-up.sql');
+  
+  return readFileAsync(filePath, {encoding: 'utf-8'})
+    .then(data => {
+      console.log('Received data:', data);
+      return db.runSql(data);
+    })
+    .then(() => {
+      const password = process.env.USER_PASSWORD || 's123';
+      const hashedPassword = bcrypt.hashSync(password, 10);
+      
+      return db.runSql(
+        'INSERT INTO public.users(user_id, name, username, password) VALUES ($1, $2, $3, $4)',
```

Comment:
if this is used as a seeding query it should have a process just like migration files there are seeder files, about writing sql in the index file its not the best way for isolating/organizing your code u need to move it to a seeder file.